### DISCUSSION TITLE
Error while starting second node

### DISCUSSION THREAD

**Discussion by shivakumarboga** [0xPolygon/polygon-edge#681] [Q&A] [answered]
Hi,

We have done cloud setup polygon-edge as mentioned in documentation. But while trying to start second node on the same server, we are getting below error. Can you please help resolve this.

[INFO]  polygon.ibft: validator key: addr=0x3D0D5EaEA0f63B0272340E7F7ce59ab6fD8F9AF8
listen tcp 127.0.0.1:9632: bind: address already in use

Is it possible to run multiple nodes on same server ?

**Comment by laviniat1996** [comment]
Hi @shivakumarboga ,

Can you please provide more info? You can use this [template](https://docs.polygon.technology/docs/edge/community/report-bug) and open an issue for this error. Thank you.

**Comment by shivakumarboga** [comment, marked as answer]
Issue is resolved. I gave different port numbers while starting other nodes on same server.
